docker-compose.v6.yml

```diff
@@ -0,0 +1,74 @@
+version: '2'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.5.2
+    container_name: elasticsearch
+    environment:
+      - cluster.name=vulnwhisperer
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - xpack.security.enabled=false
+
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    mem_limit: 8g
+    volumes:
+      - esdata1:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    restart: always
+    networks:
+      esnet:
+        aliases:
+          - elasticsearch.local
+  kibana:
+    image: docker.elastic.co/kibana/kibana:6.5.2
+    container_name: kibana
+    environment:
+      SERVER_NAME: kibana
+      ELASTICSEARCH_URL: http://elasticsearch:9200
+    ports:
+      - 5601:5601
+    depends_on:
+      - elasticsearch
+    networks:
+      esnet:
+        aliases:
+          - kibana.local
+  logstash:
+    image: docker.elastic.co/logstash/logstash:6.5.2
+    container_name: logstash
+    volumes:
+      - ./elk6/pipeline/:/usr/share/logstash/pipeline
+      - ./data/:/opt/vulnwhisperer/data
+    environment:
+      - xpack.monitoring.enabled=false
+    depends_on:
+      - elasticsearch
+    networks:
+      esnet:
+        aliases:
+          - logstash.local
+  vulnwhisperer:
+    image: hasecuritysolutions/vulnwhisperer:latest
+    container_name: vulnwhisperer
+    entrypoint: [
+                  "vuln_whisperer",
+                  "-c",
+                  "/opt/vulnwhisperer/vulnwhisperer.ini"
+    ]    
+    volumes:
+      - ./data/:/opt/vulnwhisperer/data
+      - ./elk6/vulnwhisperer.ini:/opt/vulnwhisperer/vulnwhisperer.ini
```

Comment:
Do you think it is better to have another config file with credentials inside of the docker folder instead of using the one in the configs folder?

Reply:
What would you recommend?

Reply:
I really don't know, we could leave it as it was in the old docker-compose, importing from `./configs/frameworks_example.ini`, although I actually think this is a better approach having all of the docker-compose files together. 
We will just need to make sure its very clear in the documentation that if they are using docker-compose they need to fill the credentials in this file and not in the other one :)